packages/path_provider/path_provider_windows/CHANGELOG.md

```diff
@@ -1,4 +1,4 @@
-## NEXT
+## 2.1.3
 
```

Comment:
It looks like you accidentally dropped the description of the change in your last update. (It'll need to be adjusted from the initial version, since what it's now doing is allowing compatibility with 3.0.0, not updating to it.)

Reply:
Now we should be good